pdns/dnspacket.cc

```diff
@@ -423,7 +448,10 @@ int DNSPacket::noparse(const char *mesg, size_t length)
   d_rawpacket.assign(mesg,length); 
   if(length < 12) { 
     g_log << Logger::Debug << "Ignoring packet: too short ("<<length<<" < 12) from "
-      << d_remote.toStringWithPort()<< endl;
+      << d_remote.toStringWithPort();
+    if(d_inner_remote)
+      g_log << "("<<d_inner_remote->toStringWithPort()<<")";
+    g_log << endl;
```

Comment:
Perhaps we could call `getRemoteString()`?

Reply:
In this case I felt it might be nice to keep the ports. I considered adding a `getRemoteStringWithPort` variant but this might be the only consumer. I'll ponder some more.

Reply:
I totally missed the fact that this version has the ports! Keeping it as it is makes sense to me, then, although implementing `getRemoteStringWithPort()` to make it obvious to the next reader might be good as well :)